### PR TITLE
Update mapping descriptions in Sendgrid and Twilio

### PR DESCRIPTION
**Pull request by vadimdemedes**
Addresses Matt's feedback in https://github.com/segmentio/action-destinations/pull/126 and https://github.com/segmentio/action-destinations/pull/127#pullrequestreview-745428849.

### CODE REVIEW

File: packages/destination-actions/src/destinations/personas-messaging-sendgrid/sendEmail/index.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ const action: ActionDefinition<Settings, Payload> = {
     },
     toEmail: {
       label: 'To Email',
```
Comment (mshwery) [inline packages/destination-actions/src/destinations/personas-messaging-sendgrid/sendEmail/index.ts:79]: Ah I was thinking we would update the label not necessarily the description.

File: packages/destination-actions/src/destinations/personas-messaging-twilio/sendSms/index.ts [hunk 2]
```diff
@@ -76,7 +76,7 @@ const action: ActionDefinition<Settings, Payload> = {
     },
     toNumber: {
       label: 'To Number',
-      description: 'Number to send SMS to when testing',
+      description: 'Test Phone Number',
```
Comment (mshwery) [inline packages/destination-actions/src/destinations/personas-messaging-twilio/sendSms/index.ts:79]: Same here

**Review by mshwery — COMMENTED** [review]
Looks fine but we running we would make the label more obvious. The descriptions seems ok to me.

**Comment by vadimdemedes** [comment]
Ah shoot, I thought I did just that 🤦

**Review by mshwery — APPROVED** [review]
shupppp

**Pull request merged**